Tests/WorkspaceTests/InitTests.swift

```diff
@@ -42,6 +43,8 @@ class InitTests: XCTestCase {
             XCTAssert(fs.exists(path.appending(component: "README.md")))
             XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), [])
             XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Tests")), [])
+
+            XCTAssertBuildsOnFailure(path, errMsg: BuildPlan.Error.noBuildableTarget.description)
```

Comment:
We shouldn't perform a build here, thats too expensive for this particular functionality. I think we should just assert the contents of the manifest file.